src/js/components/recipient/RecipientOverview.jsx

```diff
@@ -16,6 +17,35 @@ const propTypes = {
 };
 
 export default class RecipientOverview extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            showInfoTooltip: false
+        };
+
+        this.showTooltip = this.showTooltip.bind(this);
+        this.closeTooltip = this.closeTooltip.bind(this);
+    }
+
+    showTooltip() {
+        this.setState({
+            showInfoTooltip: true
+        }, () => {
+            const closeButton = document.querySelector('#state-overview-tooltip__close_icon');
```

Comment:
I realize we're doing this elsewhere, but we should instead be establishing a ref using `createRef` and calling `ref.focus()` on it rather than directly interfacing with the DOM.

https://codepulse.blog/how-to-focus-element-in-react/

Reply:
Not sure how to utilize the `useRef` hook or `React.createRef()` when the class `#state-overview-tooltip__close_icon` isn't tied to a specific html element in the code. Any ideas?

Reply:
Nevermind, this is no longer needed after refactoring using `TooltipWrapper`.